mmcv/runner/checkpoint.py

```diff
@@ -139,6 +122,23 @@ def get_torchvision_models():
     return model_urls
 
 
+def get_external_models():
+    mmcv_home = _get_mmcv_home()
+    default_json_path = osp.join(mmcv.__path__[0], 'urls/open_mmlab.json')
+    default_urls = load_file(default_json_path)
+    assert isinstance(default_urls, dict)
+    external_urls = dict()
+    for json_file in scandir(mmcv_home, suffix='.json'):
+        json_path = osp.join(mmcv_home, json_file)
+        urls = load_file(json_path)
+        assert set(external_urls.keys()).isdisjoint(set(urls.keys())), \
+            'External urls should not have overlapped keys'
+        print(f'Update model urls with: {json_path}')
```

Comment:
Is this printing necessary?

Reply:
We may need to inform users about non default model links/files.